aspnetcore/blazor/fundamentals/startup.md

```diff
@@ -143,7 +143,7 @@ The `loadBootResource` function can also return:
   * [Adjust the reconnection retry count and interval](xref:blazor/fundamentals/signalr#adjust-the-reconnection-retry-count-and-interval)
   * [Hide or replace the reconnection display](xref:blazor/fundamentals/signalr#hide-or-replace-the-reconnection-display)
   * [Disconnect the Blazor circuit from the client](xref:blazor/fundamentals/signalr#disconnect-the-blazor-circuit-from-the-client)
-* [Globalization and localization: Set the culture in a Blazor WebAssembly app](xref:blazor/globalization-localization#blazor-webassembly)
+* [Globalization and localization: Statically set the culture](xref:blazor/globalization-localization#statically-set-the-culture)
```

Comment:
Do you mean Blazor Assembly or Blazor Server-Side app?

Reply:
*Good point!* ..... I only tested Blazor WASM for this type of startup (`Blazor.start`). I should **_definitely_** test the scenario for Blazor Server and confirm that it works. If not, then **Yes, indeed!** ... I need to qualify this here. If this approach (`Blazor.start`) does work for Server, then I need to update over in the glob/loc article because I currently only have it there for WASM.

I'll get this checked/confirmed and updated on the next commit.

Reply:
> If this approach (Blazor.start) does work for Server

No need for Server-Side, my comment about what you mean about **Statically set**? Which blazor app type?

Reply:
Both ... The app sets the culture and the user has no opportunity (in the app or via `Accept-Languages`) to change it.

btw -- Indeed, the `Blazor.start` scenario only works for WASM. The glob/loc topic is ok (it's in the WASM pivot). I'm making an adjustment now to the *Startup* topic to call out in the cross-link to this topic that `Blazor.start` for `applicationCulture` is WASM-only.

Reply:
> The app sets the culture and the user has no opportunity (in the app or via Accept-Languages) to change it

The culture is set via the app based on the culture provider (Cookie, QueryString, ... etc) it will be change during the app is running when the user change the query string for instance

I'm not sure if I understand you correctly ;)

Reply:
Testing seemed to indicate that the app only supports what loc middleware is configured to support. If I use ...

Blazor WebAssembly ...

```javascript
Blazor.start({
  applicationCulture: 'en-US'
});
```

... or ...

```csharp
CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
CultureInfo.DefaultThreadCurrentUICulture = new CultureInfo("en-US");
```

Blazor Server ...

```csharp
app.UseRequestLocalization("en-US");
```

... and then configure Opera with `Accept-Language` with Peruvian Spanish as the top language choice, I get **US English** content. I haven't been able to force any other culture/language. However, I don't understand how loc works at a low level, so can you give an example of the query string approach that would force a different culture/language with that :point_up: type of static culture configuration. I feel like I'm missing some major key point on how the framework works here. The basic question is: *How can the dev set an app up to support a specific culture/language that a user/visitor can't change?* ... that's what I mean here about "statically set the app's culture."

Reply:
> The basic question is: How can the dev set an app up to support a specific culture/language that a user/visitor can't change? ... that's what I mean here about "statically set the app's culture."

Using supported cultures & UI cultures, but this should work for server-side blazor if I'm not wrong